x/collection/keeper/send.go

```diff
@@ -0,0 +1,135 @@
+package keeper
+
+import (
+	sdk "github.com/line/lbm-sdk/types"
+	sdkerrors "github.com/line/lbm-sdk/types/errors"
+	"github.com/line/lbm-sdk/x/collection"
+)
+
+func (k Keeper) SendCoins(ctx sdk.Context, contractID string, from, to sdk.AccAddress, amount []collection.Coin) error {
+	if err := k.subtractCoins(ctx, contractID, from, amount); err != nil {
+		return err
+	}
+	if err := k.addCoins(ctx, contractID, to, amount); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (k Keeper) addCoins(ctx sdk.Context, contractID string, address sdk.AccAddress, amount []collection.Coin) error {
+	for _, coin := range amount {
+		balance := k.GetBalance(ctx, contractID, address, coin.TokenId)
+		newBalance := balance.Add(coin.Amount)
+		k.setBalance(ctx, contractID, address, coin.TokenId, newBalance)
+
+		if err := collection.ValidateNFTID(coin.TokenId); err == nil {
+			k.setOwner(ctx, contractID, coin.TokenId, address)
+
+			// legacy
+			k.emitEventOnDescendants(ctx, contractID, coin.TokenId, collection.NewEventOperationTransferNFT)
+		}
+	}
+
+	// create account if recipient does not exist.
+	if !k.accountKeeper.HasAccount(ctx, address) {
+		k.accountKeeper.SetAccount(ctx, k.accountKeeper.NewAccountWithAddress(ctx, address))
```

Comment:
When adding a new account like this code, Please add telemetry notice like below.
https://github.com/line/lbm-sdk/blob/abac9d769cb9149b2ecd826bdfd281597d5c5dcf/x/bank/keeper/send.go#L150

Reply:
Is it necessary? Several modules of cosmos-sdk (e.g. `x/feegrant`) does not use the telemetry on the account creation.

Reply:
When I check it, It seems only `x/feegrant` module is not added. So I think we need to register PR to cosmos-sdk repo about `x/feegrant` module if possible.

Reply:
I added it.